aws_lambda_tools/clients/__init__.py

```diff
@@ -6,6 +6,16 @@
     is_dev = False
 
 if is_dev:
-    from dev_client import Client
+    try:
+        # Python 2
+        from dev_client import Client
+    except ImportError:
+        # Python 3
+        from clients.dev_client import Client
```

Comment:
would using this import break python2?

Reply:
Hey @ashstorystream I don't think it would but I now have doubts! - Lets chat offline!